Source/views/profile.hbs

```diff
@@ -34,6 +34,6 @@
 		</tr>
 	</tbody>
 </table>
-<button class="btn btn-warning">{{i18n "account.edit_profile"}}</button>
+<button class="btn btn-primary">{{i18n "account.edit_profile"}}</button>
```

Comment:
De knop werkt nu niet meer. Zie /js/profile.js

Reply:
Zorg ook dat je de account informatie netjes in een tabel zet zoals bijvoorbeeld in je edit pagina of de admin pagina.
Niet de losse tekst op de pagina